### PR TITLE
update db docs

### PR DESCRIPTION
**Pull request by JessicaGreben**
This PR does the following:
- Updates the test docs for how we now create ports for storj-sim
- Add more details to kademlia db docs
- Add steps to run storj-sim with postgres

### CODE REVIEW

File: Test-network.md [hunk 1]
```diff
@@ -54,12 +54,17 @@ Amazon S3 compatible clients.
...
-* Satellite Console starts on port `10100`
-* Storage Nodes public ports start from port `12000`
-* Storage Nodes private ports start from port `13000`
+* Satellite Console starts on port `10002`
+* Storage Nodes public ports start from port `14000`
+* Storage Nodes private ports start from port `14001`
+
+See [storj-sim network source code](https://github.com/storj/storj/blob/master/cmd/storj-sim/network.go#L36) for more details.
 
 To get access to a gateway and test your keys, you open http://127.0.0.1:9000 in a web browser.
```
Comment (stefanbenten) [inline Test-network.md]: Port 11000 right?

File: code/persistentstorage/Databases.md [hunk 2]
```diff
@@ -68,7 +67,18 @@ Reference:
...
+The kademlia bolt bucket stores the following data:
+- key: k-bucket IDs (k-buckets are groupings of nodeIDs based on prefix, read original [kademlia paper](https://pdos.csail.mit.edu/~petar/papers/maymounkov-kademlia-lncs.pdf) for details of k-buckets)
+- value: timestamp
+
+The node bolt bucket stores the following data:
+- key: nodeIDs
+- value: contact info for that node
+
+Resources:
+- [Slide deck](https://docs.google.com/presentation/d/1jY9mu8SVmVqirbo_yFIJjHWbeHrEc1NxWkFS1iauT_I/edit#slide=id.p) to a great talk [Jennifer](https://github.com/jenlij) did on storj Kademlia implementation.
```
Comment (stefanbenten) [inline code/persistentstorage/Databases.md]: Is that presentation shared with the public?

File: Test-network.md (hunk 1 above)
Comment (JessicaGreben) [reply Test-network.md]: oops missed that. thanks

File: code/persistentstorage/Databases.md (hunk 2 above)
Comment (JessicaGreben) [reply code/persistentstorage/Databases.md]: i dont think so

Comment (stefanbenten) [reply code/persistentstorage/Databases.md]: We probably dont want to add it here then. Or ask @jenlij to share it with the world 👍

Comment (JessicaGreben) [reply code/persistentstorage/Databases.md]: @jenlij is that slide deck safe to share with the world? i think it would be a great resource to link here.

Comment (jenlij) [reply code/persistentstorage/Databases.md]: https://docs.google.com/presentation/d/1jY9mu8SVmVqirbo_yFIJjHWbeHrEc1NxWkFS1iauT_I/edit?usp=sharing

Here is a shareable link!

**Pull request merged**